CHANGELOG.md

```diff
@@ -8,11 +8,13 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
-## [0.19.0] - 2021-03-19
+## [0.19.0] - 2021-03-29
 
 ### Changed
 
 - Upgrade to v0.19.0 of `go.opentelemetry.io/otel`.
+- The `go.opentelemetry.io/contrib/instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo` instrumentation now accepts a `WithCommandLoggingDisabled`, 
+   so the caller can specify whether to opt-out of tracing the mongo command. (#712)
```

Comment:
This needs to be moved to the unreleased section (a new `### Changed` section needs to be added there.

Reply:
sure thing, thanks